### PR TITLE
feat(suite-native): Add already imported account section

### LINKED ISSUE #6555: Polishing: Get warning message when trying to import same account / asset
When user tries to import account / assets she has already imported, she should get some message: This asset has been already imported

### PR DESCRIPTION
**Pull request by juriczech**
## Description
When importing accounts in onboarding, this PR ensures that user gets notified when he's importing same xpub for the second time.

## Related Issue

Resolve #6555 

## Screenshots (if appropriate):

https://user-images.githubusercontent.com/36101761/199389153-686aff91-ace9-4e5d-8d33-6df7ce8b098b.mov

### CODE REVIEW

File: suite-native/module-accounts-import/src/components/AccountImportImportedAccount.tsx [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    const handleNavigateToDashboard = () =>
+        navigation.navigate(RootStackRoutes.AppTabs, {
+            screen: AppTabsRoutes.HomeStack,
+            params: {
+                screen: HomeStackRoutes.Home,
+            },
+        });
+
+    return (
+        <AccountImportSummarySection title="Asset already imported">
```
Comment (Nodonisko) [inline suite-native/module-accounts-import/src/components/AccountImportImportedAccount.tsx]: I am still confused about naming, should there be `Account already imported?`

File: suite-native/module-accounts-import/src/components/AccountImportImportedAccount.tsx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    AccountsImportStackParamList,
+    AccountsImportStackRoutes.AccountImport,
+    RootStackParamList
+>;
+
+const contentWrapperStyle = prepareNativeStyle(() => ({
+    marginBottom: 114,
+}));
+
+export const AccountImportImportedAccount = ({ account }: AccountImportImportedAccountProps) => {
```
Comment (Nodonisko) [inline suite-native/module-accounts-import/src/components/AccountImportImportedAccount.tsx]: Can you rename this to `AccountAlreadyImported`? Current name is really confusing.

File: suite-native/module-accounts-import/src/components/AccountImportImportedAccount.tsx [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+} from '@suite-native/navigation';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+import { AccountImportSummarySection } from './AccountImportSummarySection';
+
+type AccountImportImportedAccountProps = {
+    account: Account;
+};
+
+type NavigationProp = StackToTabCompositeProps<
```
Comment (Nodonisko) [inline suite-native/module-accounts-import/src/components/AccountImportImportedAccount.tsx]: I don't think you should have this type here, this type should in `@suite-native/navigation` I think

**Review by Nodonisko — CHANGES_REQUESTED** [review]

File: suite-native/module-accounts-import/src/components/AccountImportImportedAccount.tsx (hunk 1 above)
Comment (juriczech) [reply suite-native/module-accounts-import/src/components/AccountImportImportedAccount.tsx]: This was in figma and was approved by copywriter so I just went with it. You can raise a comment if you think it should be renamed. But I guess I wouldn't wait for reply with this PR and just go with this and we can always change it if it's requested.

File: suite-native/module-accounts-import/src/components/AccountImportImportedAccount.tsx (hunk 2 above)
Comment (juriczech) [reply suite-native/module-accounts-import/src/components/AccountImportImportedAccount.tsx]: renamed

File: suite-native/module-accounts-import/src/components/AccountImportImportedAccount.tsx (hunk 3 above)
Comment (juriczech) [reply suite-native/module-accounts-import/src/components/AccountImportImportedAccount.tsx]: i will create follow up task for this and add comments to existing types (probably in separate PR to not hold this one)

**Review by Nodonisko — APPROVED** [review]

**Pull request merged**